packages/react/src/file-uploader/FileUploader.tsx

```diff
@@ -0,0 +1,31 @@
+import { FileUpload } from "@ark-ui/react";
+
+export const FileUploader = () => {
```

Comment:
let's call this `FileUpload` similar to the upstream component
```suggestion
export const FileUpload = () => {
```